deployment/ansible/group_vars/all.example

```diff
@@ -90,3 +90,9 @@ monit_allow_hosts: []
 monit_allow_user: 'test'
 monit_allow_password: 'test'
 monit_enable_web_access: yes
+
+## IP addresses used to configure the firewall
+ip_addresses_app:
```

Comment:
Shouldn't it be possible to determine these values from the inventory for a given run?